<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>802</width>
    <height>516</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>802</width>
    <height>516</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Huffman</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="compress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decompress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_original_size">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Size: N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_compressed_size">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Compressed Size: N/A</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>361</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_file_path">
       <property name="text">
        <string>File Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_lookup_table">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lookup Table</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lookup_table_display">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>361</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="compressed_stream">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>361</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_compressed_stream">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Compressed Stream</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="decompressed_stream">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_decompressed_stream">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompressed Stream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showShowInFolder_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showShowInFolder">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_decompressed_size">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompressed Size: N/A</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
